Length_of_string.c

```diff
@@ -0,0 +1,19 @@
+#include<stdio.h>
+int main()
+{
+    int i,c=0;
+    char str[10],*p;
+    printf("enter a string\n");
```

Comment:
@komal7141  write comments in this program to make it self-explainatory. 
Moreover, at the end of the script, you make take an example to explain  the working usecase.

Reply:
I did what you asked me to do.if some further improvement necessary is necessary then let me know.Thank you